tekton/images/ko/Dockerfile

```diff
@@ -17,3 +17,10 @@ LABEL maintainer "Tekton Authors <tekton-dev@googlegroups.com>"
 ENV GO111MODULE on
 RUN go get github.com/google/ko/cmd/ko@master
 
+RUN apk update
+RUN apk add --no-cache musl-dev
```

Comment:
@mnuttall can we have those all on one line (`--no-cache` means we shouldn't need `apk update` before)

```
RUN apk add --no-cache musl-dev gcc git
```

:angel:

Reply:
/mario build tekton/images/ko-gcloud ko:gcloud-mario

Reply:
I don't know if Mario listens to me : /